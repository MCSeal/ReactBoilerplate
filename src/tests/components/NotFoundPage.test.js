import React from 'react';
import NotFoundPage from '../../components/NotFound';
import { shallow } from 'enzyme';


test('not found renderer', () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot();


});

