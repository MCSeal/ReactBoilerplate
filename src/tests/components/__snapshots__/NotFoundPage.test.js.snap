// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found renderer 1`] = `
<div>
  404 not found!
  <Link
    to="/"
  >
     Go home
  </Link>
</div>
`;
